app/helpers/mentor_dashboard_helper.rb

```diff
@@ -0,0 +1,14 @@
+module MentorDashboardHelper
+  SECONDS_PER_DAY = 60 * 60 * 24
+
+  def time_ago_in_words(time)
+    case (DateTime.now - time.to_datetime) * SECONDS_PER_DAY
+    when 0..1.day
+      super(time)
+    when 1.day..2.days
+      "1 day"
+    else
```

Comment:
Since ActiveSupport is included in the project, this can use the `pluralize` inflector to simplify.
https://apidock.com/rails/String/pluralize

Reply:
Updated

Reply:
I'm actually not sure the new approach is an improvement.